google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/integration/AppEngineServerEditor.java

```diff
@@ -59,31 +63,42 @@ protected void textChanged(DocumentEvent event) {
   }
 
   private void onSdkPathChanged() {
-    if (warningMessage.getText().isEmpty()) {
-      warningMessage.setVisible(true);
-      warningMessage.setForeground(JBColor.RED);
-      warningMessage.setText(GctBundle.getString("appengine.cloudsdk.location.missing.message"));
-    } else if (!CloudSdkUtil.containsCloudSdkExecutable(mySdkHomeField.getText())) {
+    CloudSdk sdk = new CloudSdk.Builder()
+        .sdkPath(Paths.get(sdkHomeField.getText()))
+        .build();
+
+    try {
+      sdk.validateAppEngineJavaComponents();
```

Comment:
I don't think this is needed with the latest common-lib version. See `AppEngineHelper#createSdk` - the validation happens internally.